lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
     'undo the last\nblock drop\nin this pane':
-        'recupera l\'últim bloc\nque s\'hagi llençat',
+        'desfés l\'últim moviment\nde blocs',
+    'redrop':
+        'refés',
```

Comment:
... and "redrop" also moves the latest block "undrop" back again, and also undeletes any deleted block, regardless of whether it was dragged and dropped on the palette or deleted via the context menu. In essence, the upcoming release already has a better "real" undo/redo mechanism than most existing blocks environments out there.